src/plugins/console/public/application/components/help_popover.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPopover,
+  EuiTitle,
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiSpacer,
+} from '@elastic/eui';
+import { useServicesContext } from '../contexts';
+
+interface HelpPopoverProps {
+  button: any;
+  isOpen: boolean;
+  closePopover: () => void;
+  resetTour: () => void;
+}
+
+export const HelpPopover = ({ button, isOpen, closePopover, resetTour }: HelpPopoverProps) => {
+  const { docLinks } = useServicesContext();
+
+  return (
+    <EuiPopover
+      button={button}
+      isOpen={isOpen}
+      closePopover={closePopover}
+      anchorPosition="downRight"
+      buffer={[4, 4, 4, 4]}
```

Comment:
If all boundries are the same I think we can just pass the number as a parameter:

```suggestion
      buffer={4}
```

Reply:
Addressed in [472856e](https://github.com/elastic/kibana/pull/191092/commits/472856e2c32aa4525c0c88d2cdf1084d582d1c1e)